jobs/JEVS_GLOBAL_ENS_WAVE_GRID2OBS_PREP

```diff
@@ -59,7 +59,7 @@ export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${MODELNAME}/${gefs_ver})} 
 export COMINgdas=${COMINgdas:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
 
-export COMOUT=${COMOUT:-${COMIN}/${STEP}/${COMPONENT}/${RUN}}
+export COMOUT=${COMOUT:-$(compath.py ${NET}/${evs_ver})/${STEP}/${COMPONENT}/${RUN}}
```

Comment:
As noted by Alicia, she beat me to the punch, these need `-o` added. I didn't get a chance to explain it in our little lesson earlier today. But please change line 62 to: 
```
export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/${STEP}/${COMPONENT}/${RUN}}
```
I can explain why in chat tomorrow.